source/base/Profiler.ooc

```diff
@@ -65,6 +65,7 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
+		This printResults()
```

Comment:
Hmm I think not. I would expect anyone who uses `Profiler` to call `printResults` explicitly.